Storage/StorageTests/CoreData/MigrationTests.swift

```diff
@@ -3035,6 +3058,11 @@ private extension MigrationTests {
             campaign.setValue(1, forKey: "siteID")
         }
 
+        // Required since model 101
+        if modelVersion >= 101 {
+            campaign.setValue(123, forKey: "productId")
```

Comment:
`productID` is an `NSNumber`, so we should create an instance of that type instead of using Int. It would be great if you add another test case to check that in version 101, the campaign created with this method has the expected value.

Reply:
> productID is an NSNumber, so we should create an instance of that type instead of using Int.

Handled in https://github.com/woocommerce/woocommerce-ios/pull/11067/commits/e3e787d9dfaa4ff1c7eaf9fa416e34c2fb45938b

> It would be great if you add another test case to check that in version 101, the campaign created with this method has the expected value.

Handled in https://github.com/woocommerce/woocommerce-ios/pull/11067/commits/13e8b5eee392a1ad5f74e2eb7e5ee680777d34ce